internal/integration/emitter.go

```diff
@@ -6,18 +6,20 @@ package integration
 import (
 	"encoding/json"
 	"fmt"
-	"net/http"
-	"os"
+	"math"
 	"time"
 
 	"github.com/newrelic/go-telemetry-sdk/cumulative"
 	"github.com/newrelic/go-telemetry-sdk/telemetry"
+	"github.com/newrelic/nri-prometheus/internal/histogram"
+	mpb "github.com/prometheus/client_model/go"
```

Comment:
Why **mpb**? I was reviewing the code and had no idea about what it meant. In other parts of the codebase we call this import `dto`.